aten/src/THCAlloc/THCGeneral.h

```diff
@@ -0,0 +1,217 @@
+#ifndef THC_GENERAL_INC
```

Comment:
Why did `THCGeneral.h` end up in `THCAlloc`, and what is `THCAlloc`?

Reply:
I don't want THCAlloc in this PR, I removed it.